docs/posts/llms-and-data/index.qmd

```diff
@@ -0,0 +1,250 @@
+---
+title: LLMs and data
+author: "Cody Peterson"
+date: "2023-09-26"
+freeze: auto
+categories:
+    - blog
+    - ai
+    - llm
+---
+
+## Introduction
+
+While large-language models (LLMs) have been around for years, recent versions
+and innovations have made it possible to create instruction-following,
+conversational bots that can perform tasks on behalf of the user. The
+thought of using natural language to transform and analyze data is appealing.
+
+## Approaches
+
+When discussed internally at Voltron Data, we identified three primary
+approaches to applying LLMs to data that can be implemented today:
+
+1. LLM writes SQL
+2. LLM writes a UDF (Python or another language)
+3. Use LLM in a UDF (that a LLM could write)
+
```

Comment:
Can you add another sentence here to clarify that these three approaches are not necessarily the only approaches? There are surely many others, such as having the LLM generate data.

Reply:
Another suggested addition here: Say that all three of these approaches can be demonstrated using Marvin, and explain briefly what Marvin is.

Reply:
@ianmcook added a blurb if you want to review that